pom.xml

```diff
@@ -45,7 +45,7 @@
     <properties>
         <version.major>1</version.major>
         <version.minor>3</version.minor>
-        <version.patch>13</version.patch>
```

Comment:
The current released version of the engine is 1.3.12, so let's keep the version here at 1.3.13